app/bundles/LeadBundle/Views/Company/company.html.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
```

Comment:
Why copyright 2014 ? I think it should be 2020, but better to ask Mautic people :-)